src/Message/ApplePayAuthorizeResponse.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Omnipay\Vindicia\Message;
+
+class ApplePayAuthorizeResponse extends Response
+{
+    /**
+     * Was the response successful?
+     *
+     * @return boolean
+     */
+    public function isSuccessful()
+    {
+        /**
+         * @var boolean
+         */
+        return ($this->data['statusCode'] == 200);
+    }
+
+    /**
+     * The entire Apple Pay payment session object to be used to validate the merchant.
+     * This should be passed to the front end.
+     *
+     * @return static
+     */
+    public function getPaymentSessionObject()
+    {
+        /**
+         * @var static
+         */
+        return (empty($this->data['body'])) ? '' : json_encode($this->data['body']);
+    }
+
+    /**
+     * The human-readable status code.
+     *
+     * @return static
+     */
+    public function getReason()
+    {
+        /**
+         * @var static
+         */
+        return $this->data['reasonPhrase'];
+    }
+
+    /**
+     * The numerical status code.
+     *
+     * @return static
+     */
+    public function getStatusCode()
```

Comment:
Omnipay calls this "code", not "statusCode". see https://github.com/vimeo/omnipay-vindicia/blob/master/src/Message/Response.php